prover/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "wormhole-prover"
+version.workspace = true
+edition.workspace = true
+authors.workspace = true
+description.workspace = true
+license.workspace = true
+
+[dependencies]
+anyhow.workspace = true
+plonky2.workspace = true
+poseidon-resonance.workspace = true
+wormhole-circuit = { path = "../circuit" }
+
+[features]
+testing = ["wormhole-circuit/testing"]
+
+[[bench]]
+name = "prover_bench"
+harness = false
+required-features = ["testing"]
+
+[dev-dependencies]
+criterion = { workspace = true, features = ["html_reports"] }
```

Comment:
Why do we enable the `html_reports` feature in this crate and the verifier, is it needed?

Reply:
lol no clue, will remove this